src/components/PersonLink.tsx

```diff
@@ -0,0 +1,25 @@
+import classNames from 'classnames';
+import { Link, useSearchParams } from 'react-router-dom';
+import { Person } from '../types';
+
+type Props = {
+  person: Person,
+};
+
+export const PersonLink: React.FC<Props> = ({ person }) => {
+  const [searchParams] = useSearchParams();
+
+  return (
+    <Link
+      to={{
+        pathname: `/people/${person.slug}`,
+        search: searchParams.toString(),
+      }}
+      className={classNames({
+        'has-text-danger': person.sex === 'f',
```

Comment:
Create a enum for this magic word and use them everywhere

Reply:
it will be the best enum